src/pat/contentbrowser/src/SelectedItems.svelte

```diff
@@ -85,6 +92,10 @@
         selectedItemsNode.dispatchEvent(events.change_event());
     }
 
+    function LoadSelectedItemComponent(node, props) {
+        const component = new RegisteredSelectedItem.component({target: node, props: props});
```

Comment:
```suggestion
        const component = plone_registry.getComponent({
            name: "pat-contentbrowser.SelectedItem",
            dependencies: [props.item["@type"]]
        }).component;
        return component({target: node, props: props});
```
NOTE: untested pseudo code